<!DOCTYPE html>
<html>

<head>
  <!-- Basic -->
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <!-- Mobile Metas -->
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
  <!-- Site Metas -->
  <meta name="keywords" content="" />
  <meta name="description" content="" />
  <meta name="author" content="" />

  <title>Results</title>

  <!-- slider stylesheet -->
  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.1.3/assets/owl.carousel.min.css" />

  <!-- bootstrap core css -->
  <link rel="stylesheet" type="text/css" href="css/bootstrap.css" />

  <!-- fonts style -->
  <link href="https://fonts.googleapis.com/css?family=Baloo+Chettan|Dosis:400,600,700|Poppins:400,600,700&display=swap" rel="stylesheet" />
  <!-- Custom styles for this template -->
  <link href="css/style.css" rel="stylesheet" />
  <!-- responsive style -->
  <link href="css/responsive.css" rel="stylesheet" />
</head>

<body>
  <div class="hero_area">
<!--    &lt;!&ndash; header section strats &ndash;&gt;-->
<!--    <div class="brand_box">-->
<!--      <a class="navbar-brand" href="index.html">-->
<!--        <span>-->
<!--          Ninom-->
<!--        </span>-->
<!--      </a>-->
<!--    </div>-->
<!--    &lt;!&ndash; end header section &ndash;&gt;-->
  </div>
<!-- slider section -->
<section class=" slider_section position-relative">
  <div id="carouselExampleControls" class="carousel slide " data-ride="carousel">
    <div class="carousel-inner">
      <div class="carousel-item active">
        <div class="img-box">
          <img src="images/bgpic1.png" alt="">
        </div>
      </div>
      <!--          <div class="carousel-item">-->
      <!--            <div class="img-box">-->
      <!--              <img src="images/bgpic1.png" alt="">-->
      <!--            </div>-->
      <!--          </div>-->
      <!--          <div class="carousel-item">-->
      <!--            <div class="img-box">-->
      <!--              <img src="images/bgpic1.png" alt="">-->
      <!--            </div>-->
      <!--          </div>-->
    </div>
    <!--        <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">-->
    <!--          <span class="sr-only">Previous</span>-->
    <!--        </a>-->
    <!--        <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">-->
    <!--          <span class="sr-only">Next</span>-->
    <!--        </a>-->
  </div>
</section>
<!-- end slider section -->

  <!-- nav section -->
<section class="nav_section">
  <div class="container">
    <div class="custom_nav2">
      <nav class="navbar navbar-expand custom_nav-container ">
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <div class="d-flex  flex-column flex-lg-row align-items-center">
            <ul class="navbar-nav  ">
              <li class="nav-item active" style="font-size: 20px">
                <a class="nav-link" href="project.html">Project </a>
              </li>
              <li class="nav-item active" style="font-size: 20px">
                <a class="nav-link" href="introduction.html">Introduction <span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item" style="font-size: 20px">
                <a class="nav-link" href="results.html">Results </a>
              </li>
              <li class="nav-item" style="font-size: 20px">
                <a class="nav-link" href="poster.html">Poster </a>
              </li>
              <li class="nav-item" style="font-size: 20px">
                <a class="nav-link" href="index.html">Home </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </div>
  </div>
</section>

  <!-- end nav section -->
  <!-- about section -->
  <section class="about_section layout_padding">
    <div>
      <h1 align="center" style="font-size: 60px">Results and Discussions </h1>
    </div>
    <div class="container-fluid">
          <div align="center">
            <hr>
            <h2 id="result1">
              Substrate selection of P450BM3
            </h2>
            <img src="images/图片3.png" alt="">
            <h5 align="center">Figure 2. Biochemical assays of P450BM3-holoenzyme (Heme-FMN-FAD) using 4-nitrophenyl acetate (1) or 4-nitrophenyl butyrate (3) as substrate. (i) 4-nitrophenyl acetate (1). (ii) 1 + NADPH + P450BM3-holoenzyme (5 μM, reaction for 2 h). (iii) 4-nitrophenyl butyrate (3). (iv) 3 + NADPH + P450BM3-holoenzyme (5 μM, reaction for 2 h).</h5>
          </div>
          <div class="detail-box">
            <p align="left" style="font-size: 25px">
             The P450BM3 gene was synthesized per the DNA sequence from GenBank (ID: KX768143.1). We then amplified the sequences encoding P450BM3-holoenzyme and P450BM3-heme domain, and cloned them into pET28a, respectively. The soluble form of P450BM3-holoenzyme (119 KDa) and P450BM3-heme domain (55 KDa) were obtained in Escherichia coli BL21 (DE3) (Fig S1). To find the possible substrates of P450BM3, we tested the catalytic activities of P450BM3 toward different small molecule compounds, including the benzene ring-containing compounds 3,4-dichloroaniline, 3,4-dichlorothiophenol, 4-nitrophenyl acetate, 4-nitrophenyl butyrate and 7-hydroxycoumarin, and the cholesterol-lowering compound lovastatin, anti-tumor compound epothilone D and the anti-oxidant compounds resveratrol and polydatin (Fig S2). The reactions were set up using P450BM3-holoenzyme (Heme-FMN-FAD) and NADPH as cofactor or P450BM3-heme domain and H2O2 as cofactor. As shown in Fig 2, in the presence of NADPH, P450BM3-holoenzyme (Heme-FMN-FAD) could transform 4-nitrophenyl acetate (1) to 3 as well as 4-nitrophenyl butyryl (2) to 4. Therefore, we choose 4-nitrophenyl acetate and 4-nitrophenyl butyryl as the substrates for P450BM3 for further study.
            </p>
          </div>
    </div>
  </section>
<hr>
<section class="about_section layout_padding">
  <div class="container-fluid">
        <div align="center">
          <h2 id="result2">
            Identification of the candidate residues of P450BM3 for directed evolution
          </h2>
          <img src="images/图片4.png" alt="">
          <h5 align="center">Figure 3. Electrostatic surface potential analysis of P450BM3 (PDB id: 4ZFA).</h5>
        </div>
        <div class="detail-box">
          <p align="left" style="font-size: 25px">
            The crystal structure of P450BM3 was identified and deposited in PDB database (PDB id: 4ZFA). The protein surface was reported to be vital important for the enzyme to adapt to a particular environment. To probe the possible residues on the surface that may be closely related to the optimum pH of P450BM3, we used APBS to evaluate the electrostatic surface potentials of P450BM3. As shown in Fig 3, the “front” side of P450BM3 containing αA, αE and αH, is electronegative, while the “back” side of P450BM3 containing αD, αL and αJ, is electropositive. We then analyzed the amino acid sequences of these charged regions. “P18LLNTDKPVQALMKIAD34” and “L418KHFDFED425” constitute the negatively charged regions, and “H236MLNGKDPE244” and “D338TVLGGEYPL347” compose the positively charged regions. As it was reported that ionizable amino acids might have greater effect on pH optima than aliphatic ones, we choose the 13 amino acids with charged side chains (Asp23, Lys24, Lys31, Asp34, Lys419, Asp422, Glu424, Asp425, Lys241, Asp242, Glu244, Asp338 and Glu344) for further evolution study.
          </p>
        </div>
      </div>
</section>
<hr>
  <section class="about_section layout_padding">
    <div class="container-fluid">
      <div align="center">
        <h2 id="result3">
         Structure-based virtual screening of P450BM3
        </h2>
        <img src="images/图片7.png" alt="">
        <h5 align="center">Table 1. Mutation sites of the top-ranking 7 mutants with enhanced stabilities.</h5>
      </div>
      <div class="detail-box">
        <p align="left" style="font-size: 25px">
          We constructed a virtual enzyme library of 247 saturated mutants of the 13 selected amino acids (Asp23, Lys24, Lys31, Asp34, Lys419, Asp422, Glu424, Asp425, Lys241, Asp242, Glu244, Asp338 and Glu344). To select the possible hints with higher stabilities, we used DynaMut website (https://biosig.lab.uq.edu.au/dynamut/) to measure the protein stability changes of these mutants. As shown in Table 1, the top-ranking 7 mutants with enhanced stabilities were selected for further study.        </p>
      </div>
    </div>
  </section>
<hr>
  <section class="about_section layout_padding">
    <div class="container-fluid">
      <div align="center">
        <h2 id="result4">
         Construction and activity tests of mutants
        </h2>
        <img src="images/图片5.png" alt="">
        <h5 align="center">Figure 4. The relative activities of the crude enzymes of BmmI and its variants toward 4-nitrophenyl acetate (A) and 4-nitrophenyl butyrate (B).</h5>
      </div>
      <div class="detail-box">
        <p align="left" style="font-size: 25px">
          We then constructed the 7 mutants K24I, D242L, D242V, D425M, D422W, D422Y and D242P. Reverse complementary primers with mutation sites were designed to amplify the linear expression plasmids (pET28a/ P450BM3-holo) with mutated sequences (Table S2). The resulting linear plasmids were ligated to circular molecules by seamless cloning, and then introduced into E. coli BL21 (DE3). The expression strains were individually cultured, followed by addition of 0.2 mM IPTG to induce protein expression. Crude enzymes from each expression strain was incubated with NADPH and 4-nitrophenyl acetate or 4-nitrophenyl butyrate, respectively, for 2 h. The generation of products was detected by HPLC. As shown in Fig 4, compared to that of the wild-type P450BM3, the catalytic activities of K24I toward 4-nitrophenyl acetate and 4-nitrophenyl butyrate increased by ~1-fold and 0.4-fold, respectively; the catalytic activities of D422W toward 4-nitrophenyl acetate and 4-nitrophenyl butyrate increased by ~1-fold and 0.2-fold, respectively (Fig 4).      </div>
    </div>
  </section>
<hr>
  <section class="about_section layout_padding">
    <div class="container-fluid">
      <div align="center">
        <h2 id="result5">
          In vitro characterization of the P450BM3 variants K24I and D423W in different pH
        </h2>
<!--        <img src="images/result5.jpg" alt="">-->
        <img src="images/图片6.png" alt="">
        <h5 align="center">Figure 5. The relative activities of the wild-type-P450BM3, K24I and D422W toward 4-nitrophenyl acetate (A) and 4-nitrophenyl butyrate (B) in different pH buffers.</h5>
      </div>
      <div class="detail-box">
        <p align="left" style="font-size: 25px">
          Furthermore, we purified the P450BM3 variants K24I and D423W (Fig S3), and tested the catalytic activities of the mutants and wild-type enzyme in acidic (Na2HPO3-Citric acid, pH=6.0), neutral (Na2HPO3-NaH2PO3, pH=7.5) and alkaline (Glycine-NaOH, pH=9.0) buffer, respectively. As shown in Fig 5A, in acidic buffer (pH=6.0), the catalytic activities of K24I and D422W increased by ~0.5-fold and 0.4-fold, respectively, compared to wild-type enzyme. These results indicated that the mutations of Lys24 and Asp422 could effectively improve the catalytic activities of P450BM3 in acidic conditions.
    </div>
    </div>
  </section>
<hr>
  <section class="about_section layout_padding">
    <div class="container-fluid">
      <div align="center">
        <h2 id="conclusion">
          Conclusion
        </h2>
<!--        <img src="images/result4.jpg" alt="">-->
<!--        <h5 align="center">Figure 5. The relative activities of the crude enzymes of BmmI and its variants.</h5>-->
      </div>
      <div class="detail-box">
        <p align="left" style="font-size: 25px">
          P450BM3 is a natural fusion enzyme comprising a heme-binding domain and a NADPH-P450 reductase domain. Its high activity and catalytic self-sufficiency make P450BM3 an excellent platform for biocatalysis. In this project, we analyzed the possible residues on the surface that may be closely related to the optimum pH of P450BM3 and constructed a virtual of 380 saturated mutants. The top7 mutants with enhanced stabilities were selected by using DynaMut website calculation. The variants K24I and D422W exhibited ~0.5-fold and 0.4-fold increased activities toward 4-nitrophenyl acetate in acidic condition (pH=6.0).
    </div>
    </div>
  </section>

  <!-- end about section -->

  <!-- info section -->

<section class="info_section layout_padding">
  <div class="container">
    <div class="info_logo">
      <h2>
        OUC-Marine Drugs
      </h2>
    </div>
    <div class="info_contact">
      <div class="row">
        <div class="col-md-4">
          <a href="">
            <img src="images/location.png" alt="">
            <span>
                Ocean University of China, Qingdao 266003, China
              </span>
          </a>
        </div>
        <div class="col-md-4">
          <a href="">
              <span>
              </span>
          </a>
        </div>
        <div class="col-md-4">
          <a href="">
            <img src="images/mail.png" alt="">
            <span>
                xiaofei3450@ouc.edu.cn
              </span>
          </a>
        </div>
      </div>
    </div>
    <div class="row">
    </div>

  </div>
</section>

  <!-- end info section -->


  <!-- footer section -->
  <section class="container-fluid footer_section">
    <p>
      &copy; <span id="displayYear"></span> All Rights Reserved. Design by Ocean University of China
    </p>
  </section>
  <!-- footer section -->


  <script type="text/javascript" src="js/jquery-3.4.1.min.js"></script>
  <script type="text/javascript" src="js/bootstrap.js"></script>
  <script type="text/javascript" src="js/custom.js"></script>
</body>

</html>
